alf/utils/losses.py

```diff
@@ -51,10 +51,50 @@ def huber_function(x: torch.Tensor, delta: float = 1.0):
 
     Args:
         x: difference between the observed and predicted values
-        delta: the threshold at which to change between delta-scaled 
+        delta: the threshold at which to change between delta-scaled
             L1 and L2 loss, must be positive. Default value is 1.0
     Returns:
         Huber function (Tensor)
     """
     return torch.where(x.abs() <= delta, 0.5 * x**2,
                        delta * (x.abs() - 0.5 * delta))
+
+
+@alf.configurable
+def multi_quantile_huber_loss(quantiles: torch.Tensor,
+                              target: torch.Tensor,
+                              delta: float = 0.1) -> torch.Tensor:
+    """Multi-quantile Huber loss
+
+    The loss for simultaneous multiple quantile regression. The number of quantiles
+    n is ``quantiles.shape[-1]``. ``quantiles[..., k]`` is the quantile value
+    estimation for quantile :math:`(k + 0.5) / n`. For each prediction, there
+    can be one or multiple target values.
+
+    This loss is described in the following paper:
+
+    `Dabney et. al. Distributional Reinforcement Learning with Quantile Regression
+    <https://www.aaai.org/ocs/index.php/AAAI/AAAI18/paper/viewFile/17184/16590>`_
+
+    Args:
+        quantiles: batch_shape + [num_quantiles,]
+        target: batch_shape or batch_shape + [num_targets, ]
+        delta: the smoothness parameter for huber loss (larger means smoother).
```

Comment:
Do you want to briefly describe the rationale of setting the default delta to be 0.1 rather than the commonly used 1.0?